engine/rsrc/fontmap

```diff
@@ -0,0 +1 @@
STIXIntegralsUpSm-Bold=STIXIntegralsUpSm-Bold
STIXIntegralsUpSm-Regular=STIXIntegralsUpSm-Regular
STIXNonUnicode-BoldItalic=STIXNonUnicode-BoldItalic
STIXNonUnicode-Italic=STIXNonUnicode-Italic
STIXNonUnicode-Regular=STIXNonUnicode-Regular
STIXSizeFiveSym-Regular=STIXSizeFiveSym-Regular
STIXSizeFourSym-Bold=STIXSizeFourSym-Bold
STIXSizeFourSym-Regular=STIXSizeFourSym-Regular
STIXSizeOneSym-Bold=STIXSizeOneSym-Bold
STIXSizeOneSym-Regular=STIXSizeOneSym-Regular
STIXSizeThreeSym-Bold=STIXSizeThreeSym-Bold
STIXSizeThreeSym-Regular=STIXSizeThreeSym-Regular
STIXSizeTwoSym-Bold=STIXSizeTwoSym-Bold
STIXSizeTwoSym-Regular=STIXSizeTwoSym-Regular
STIXVariants-Bold=STIXVariants-Bold
STIXVariants-Regular=STIXVariants-Regular
STKaiti=STKaiti
STSong=STSong
STXihei=STXihei
Sukhumvit Set Light=SukhumvitSet-Light
Sukhumvit Set Medium=SukhumvitSet-Medium
Sukhumvit Set Semi Bold=SukhumvitSet-SemiBold
Sukhumvit Set Text=SukhumvitSet-Text
Sukhumvit Set Thin=SukhumvitSet-Thin
Superclarendon Black Italic=Superclarendon-BlackItalic
Superclarendon Black=Superclarendon-Black
Superclarendon Bold Italic=Superclarendon-BoldItalic
Superclarendon Bold=Superclarendon-Bold
Superclarendon Italic=Superclarendon-Italic
Superclarendon Light Italic=Superclarendon-LightItalic
Superclarendon Light=Superclarendon-Light
Superclarendon Regular=Superclarendon-Regular
Symbol=Symbol
Tahoma Negreta=Tahoma-Bold
Tahoma=Tahoma
Tamil MN Bold=TamilMN-Bold
Tamil MN=TamilMN
Tamil Sangam MN Bold=TamilSangamMN-Bold
Tamil Sangam MN=TamilSangamMN
Telugu MN Bold=TeluguMN-Bold
Telugu MN=TeluguMN
Telugu Sangam MN Bold=TeluguSangamMN-Bold
Telugu Sangam MN=TeluguSangamMN
Thonburi Bold=Thonburi-Bold
Thonburi Light=Thonburi-Light
Thonburi=Thonburi
Times Bold Italic=Times-BoldItalic
Times Bold=Times-Bold
Times Italic=Times-Italic
Times New Roman Bold Italic=TimesNewRomanPS-BoldItalicMT
Times New Roman Italic=TimesNewRomanPS-ItalicMT
Times New Roman=TimesNewRomanPSMT
Times Roman=Times-Roman
TingRoman=TingRoman
Trattatello=Trattatello
Trebuchet MS Bold Italic=Trebuchet-BoldItalic
Trebuchet MS Bold=TrebuchetMS-Bold
Trebuchet MS Italic=TrebuchetMS-Italic
Trebuchet MS=TrebuchetMS
Verdana Bold Italic=Verdana-BoldItalic
Verdana Bold=Verdana-Bold
Verdana Italic=Verdana-Italic
Verdana=Verdana
Waseem Light=WaseemLight
Waseem Regular=Waseem
Waseem=Waseem
Wawati TC Regular=DFWaWaTC-W5
Webdings=Webdings
Weibei SC Bold=Weibei-SC-Bold
Weibei TC Bold=Weibei-TC-Bold
Wingdings 2=Wingdings2
Wingdings 3=Wingdings3
Wingdings=Wingdings-Regular
Xingkai SC Bold=STXingkai-SC-Bold
Xingkai SC Light=STXingkai-SC-Light
Yuanti SC Bold=STYuanti-SC-Bold
Yuanti SC Light=STYuanti-SC-Light
Yuanti SC Regular=STYuanti-SC-Regular
YuGothic Bold=YuGo-Bold
YuGothic Medium=YuGo-Medium
YuMincho Demibold=YuMin-Demibold
YuMincho Medium=YuMin-Medium
Yuppy SC Regular=YuppySC-Regular
Yuppy TC Regular=YuppyTC-Regular
Zapf Dingbats=ZapfDingbatsITC
Zapfino=Zapfino
Zawgyi-One=Zawgyi-OnePalatino Italic=Palatino-Italic

```

Comment:
I think there are line-ending issues with this file... Also there is a missing return after Zawgyi-OnePaletino